pkg/metrics/metrics.go

```diff
@@ -101,6 +115,19 @@ func (mm *MetricsManager) RecordOperationErrorMetrics(
 	klog.Infof("Recorded PDCSI operation error code: %q", errCode)
 }
 
+func (mm *MetricsManager) RecordMountErrorMetric(fs_format string, err error) {
+	mntErr := &mount.MountError{}
+	if errors.As(err, mntErr) {
+		mountErrorMetric.WithLabelValues(pdcsiDriverName, fs_format, string(mntErr.Type)).Inc()
```

Comment:
Is this error an count? How do we calculate the SLOs if we do not have the success rate as well?